packages/editor-ui/src/utils/apiUtils.ts

```diff
@@ -62,7 +62,11 @@ async function request(config: {
 		baseURL,
 		headers,
 	};
-	if (import.meta.env.NODE_ENV !== 'production' && !baseURL.includes('api.n8n.io')) {
+	if (
+		import.meta.env.NODE_ENV !== 'production' &&
+		!baseURL.includes('api.n8n.io') &&
+		!baseURL.includes('n8n.cloud')
```

Comment:
why is this needed?

Reply:
CORS. We cannot send requests to another domain with withCredentials = true